_about/notes/0.5.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Istio 0.5
+
+order: 96
+
+layout: about
+type: markdown
+---
+{% include home.html %}
+
+In addition to the usual pile of bug fixes and perf improvements, this release includes the new or 
```

Comment:
May be a happy new year?

Reply:
Getting kind of late for that, don't you think?